core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java

```diff
@@ -73,6 +76,8 @@ protected BoundStatement bind(PreparedStatement prepared, Operation operation) {
     checkArgument(operation, Get.class, Scan.class);
     if (operation instanceof Get) {
       return bind(prepared.bind(), (Get) operation);
+    } else if (operation instanceof ScanAll) {
+      return bind(prepared.bind(), (ScanAll) operation);
```

Comment:
It looks like we don't need this `else if` block for now. But it might be good to have it for consistency with the `prepare()`. What do you think?

Reply:
Same here, I think we can remove the `else if` block for ScanAll.

Reply:
Sorry I did not think this through as well. 
Calling the `bind()` method for `ScanAll` is not required as there are no columns to bind. 

Eventually, we could still keep the `else if` block `ScanAll` but simplify it by the following. 
```
protected BoundStatement bind(PreparedStatement prepared, Operation operation) {
    checkArgument(operation, Get.class, Scan.class, ScanAll.class);
    if (operation instanceof Get) {
      return bind(prepared.bind(), (Get) operation);
    } else if (operation instanceof ScanAll) {
      return prepared.bind();
    } else {
      return bind(prepared.bind(), (Scan) operation);
    }
  }
  ```
  What do you think ?

Reply:
Sounds good!